src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonChoiceType.php

```diff
@@ -107,4 +100,25 @@ public function getBlockPrefix()
     {
         return 'sylius_taxon_choice';
     }
+
+    /**
+     * @param null|string   $root
+     * @param null|callable $filter
+     *
+     * @return array|TaxonInterface[]
+     */
+    private function getTaxons($root = null, $filter = null)
+    {
+        if (null !== $root) {
+            $taxons = $this->taxonRepository->findChildren($root, true);
+        } else {
```

Comment:
I know that another extraction would not be a good option (two methods for a simple case), but `else` does not look good too :/ How could we make it shiny and perfect?